18F/identity-idp
# LG-14007: handle redirect from socure

## 🎫 Ticket

Link to the relevant ticket:
[LG-14007](https://cm-jira.usa.gov/browse/LG-14007)

## 🛠 Summary of changes

Added a new GET route for the socure callback and pass that in on the request for the document capture app url. Upon receiving that GET, we collect the results from the DocumentCaptureSession, then if successful, redirect to the SSN step.

If not successful, it currently redirects back to the socure document capture page.

## 📜 Testing Plan

Currently deployed to a test environment:
1. Create an account/sign in
2. Go to `/verify`
3. Go through IdV until you come back from Socure
4. Verify that you land on the SSN page (assuming successful doc capture)
